targets/psa_switch/psa_switch.cpp

```diff
@@ -337,17 +420,57 @@ PsaSwitch::ingress_thread() {
     ingress_mau->apply(packet.get());
     packet->reset_exit();
 
-    // prioritize dropping if marked as such - do not move below other checks
-    const auto &f_drop = phv->get_field("psa_ingress_output_metadata.drop");
-    if (f_drop.get_int()) {
+    const auto &f_ig_cos = phv->get_field("psa_ingress_output_metadata.class_of_service");
+    const auto ig_cos = f_ig_cos.get_uint();
+
+    // ingress cloning - sent in their pre-ingress-thread state to either an egress
+    //                   port or a multicast group
```

Comment:
I am not sure I understand this comment. What does "pre-ingress-thread state" mean?

Reply:
Hmm I meant to say "packet as it was before it was picked up by the ingress thread" - but that seems incorrect now that I look at the PSA spec again. 

I changed this comment to say "each cloned packet is a copy of the packet as it entered the ingress parser" - what the PSA spec says.